utils/frame/sub-du/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "sub-du"
+version = "0.1.0"
+authors = ["kianenigma <kian.peymani@gmail.com>"]
+edition = "2021"
+
+[dependencies]
+codec = { package = "parity-scale-codec", version = "3.0.0" }
+async-std = { version = "1.12.0", features = ["attributes"] }
```

Comment:
I suggest that we switch async-std to tokio, jsonrpsee is tightly coupled to tokio otherwise you have to enable the `tokio1` feature on async std.

Perhaps that feature is leaked from the substrate workspace anyway :P